src/CustomDropdown.xml

```diff
@@ -71,6 +71,10 @@
                 <attributeTypes>
                     <attributeType name="String"/>
                 </attributeTypes>
+            </property>
+            <property key="pageSize" type="integer" required="true" defaultValue="10" >
+                <caption>Page size for pagination</caption>
+                <description>Number of records retrived per page</description>
```

Comment:
Nice, please also include a boolean that can turn on/off the whole pagination feature.

Reply:
We discussed this and instead of new attribute it's better to make pageSize optional, which would mean pagination is disabled.